asteroid/dsp/overlap_add.py

```diff
@@ -113,7 +113,7 @@ def ola_forward(self, x):
                 frame = _reorder_sources(frame, out[-1], n_src, self.window_size, self.hop_size)
 
             if self.use_window:
-                frame = frame * self.window
+                frame = frame * self.window.to(frame)
```

Comment:
I didn't know you could pass a tensor directly.

Reply:
Yeah or call frame.device. 
If you pass a tensor it also casts to the same type